File: apiForMagazyn/src/main/java/org/apiForMagazyn/api/tow/Tow.java
package org.apiForMagazyn.api.tow;

import lombok.Getter;
import lombok.Setter;
import org.apiForMagazyn.api.prod.Prod;

import javax.persistence.*;
import javax.validation.constraints.Size;

@Entity
@Table
@Setter
@Getter
public class Tow {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idTow;

    @Size(max = 50)
    private String nrSer;

    @Size(max = 50)
    private String name;

    @Size(max = 20)
    private String dim;

    @OneToOne
    @JoinColumn(name="idProd")
    private Prod prod;
}

File: Magazyn/Magazyn/Tables/Tow.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Magazyn.Tables
{
    class Tow
    {
        public int idTow { set; get; }
        public string name { set; get; }
        public string nrSer { set; get; }
        public string dim { set; get; }
        public Prod prod { set; get; }
    }
}
